webpack/scenes/Tasks/TaskActions.js

```diff
@@ -80,9 +83,10 @@ export const pollBulkSearch = (extendedParams = {}, interval, orgId) =>
       }
     };
     const { id, loading } = getState().katello.organization;
-
     if (id === orgId && !loading) {
-      return dispatch(bulkSearch(extendedParams)).then(triggerPolling);
+      const dispatchedAction = await dispatch(await bulkSearch(extendedParams));
```

Comment:
I don't like the double await as I am afraid it will confuse newcomers.

Reply:
@sharvit Ok I will expand it, I agree it will be more readable as two lines.